app/models/meetup.rb

```diff
@@ -0,0 +1,95 @@
+class Meetup
```

Comment:
We will want to move this file to the `lib` directory, as `Meetup` is not a database backed model, inheriting from `ApplicationRecord `.